### DISCUSSION TITLE
ace: Update to ACE 8.0.2

### DISCUSSION THREAD

**Discussion by jwillemsen** [Homebrew/discussions#5791] [Writing Formulae/Casks] [answered]
### Output of `brew config`

```shell
=
```

### Output of `brew doctor`

```shell
-
```

### Description of issue

ace 8.0.2 is now available, see https://github.com/DOCGroup/ACE_TAO/releases/tag/ACE%2BTAO-8_0_2

**Comment by SMillerDev** [comment, marked as answer]
https://docs.brew.sh/Updating-Software-in-Homebrew
